.github/oas.yml

```
@@ -0,0 +1,29 @@
+
+name: OAS
+
+on:
+  push:
+    branches:
+      - main
+      - stable/*
+    tags:
+      - '*'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  oas:
+    name: Checks
+    uses: maykinmedia/open-api-workflows/.github/workflows/oas.yml@refactor/reusable-actions
```

I added this change to v5, so let's use that
```suggestion
    uses: maykinmedia/open-api-workflows/.github/workflows/oas.yml@v5```